app/views/layouts/application.html.erb

```diff
@@ -1,7 +1,14 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>Wave - Content from your favorite music artists</title>
+    <title>Wave - One app for all your music artists' content</title>
+    <meta name="description" content="Use Wave to follow all content from your favorite music artists in one app.">
+    <meta charset="utf-8">
+    <meta property="og:title" content="Wave - One app for all your music artists' content">
+    <meta property="og:image" content="wave-fb">
+    <meta property="og:description" content="Use Wave to follow all content from your favorite music artists in one app.">
+    <meta property="og:site_name" content="Wave"/>
```

Comment:
Instead of having all of the text in the view, can you do it in the config/meta.yml file, I followed the turorial and it uses embedded ruby in application.html.erb, have a look at my pr for more details. Thanks.

Reply:
oh, wait. Nevermind